content/references/markdown/index.md

```diff
@@ -4,12 +4,148 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```code
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+```code
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+###### This is &lt;h6&gt;
+```
+
+
+
+## Code blocks
+
+To get text displayed without the renderer playing with the indentation or whitespaces, code blocks
+
+### Inline code blocks
+
+To highlight code  ` \`inline code block\` `
+
+
+## Lists
+
+### Unordered lists:
+
+```code
+* item 1
+* item 2
+    * subitem 2.1
+    * subitem 2.2
+```
+
+You can use different marks to get same result, like `+` or `-`.
+
+### Ordered lists:
+
+The numbering is automatic, hence you cannot mix different types of paragraphs or items if you want to keep the numbering correct.
```

Comment:
I don't think we should include this for this basic syntax cheat sheet page.

Reply:
removed